packages/menu-bar/theme/lumo/vaadin-menu-bar-styles.js

```diff
@@ -4,12 +4,12 @@ import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themab
 registerStyles(
   'vaadin-menu-bar',
   css`
-    :host([has-single-button]) [part$='button'] {
+    :host([has-single-button]) ::slotted(vaadin-menu-bar-button) {
       border-radius: var(--lumo-border-radius-m);
     }
 
-    :host([theme~='end-aligned']) [part$='button']:first-child,
-    :host([theme~='end-aligned'][has-single-button]) [part$='button'] {
+    :host([theme~='end-aligned']) ::slotted(vaadin-menu-bar-button:first-child),
```

Comment:
Tested `theme="end-aligned"` with the menu bar dev page, which has a slotted tooltip:
```
    <vaadin-menu-bar theme="end-aligned">
      <vaadin-tooltip slot="tooltip" hover-delay="500" hide-delay="500"></vaadin-tooltip>
    </vaadin-menu-bar>
```
The selector is not applied in that case.

Reply:
`first-of-type` would work instead, assuming that the container may only contain `vaadin-menu-bar-button` instances.

Apart from that I don't understand why the selector breaks due to the tooltip, the tooltip has its own slot, and I can see that the slot exists.

Reply:
Thanks, changed to use `:first-of-type` instead.